addons/payment/models/payment_token.py

```diff
@@ -125,6 +125,19 @@ def _build_display_name(self, *args, max_length=34, should_pad=True, **kwargs):
             display_name = self.payment_details[-max_length:] if max_length > 0 else ''
         return display_name
 
+    def _get_available_tokens(self, *args, **kwargs):
```

Comment:
Move this method before `_build_display_name` as it's used way earlier in the flow.